src/main/java/domain/Astar.java

```diff
@@ -0,0 +1,180 @@
+package domain;
+
+import bootstrap.Driver;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.swing.*;
+import java.awt.*;
+import java.util.*;
+import java.util.List;
+
```

Comment:
Please give the run complexity of findPath() ? 

Could be useful to put in a /* comment section */